streaming/src/test/scala/org/apache/spark/streamingtest/LocalStreamingContext.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.streamingtest
```

Comment:
Why `streamingtest` vs `streaming.testutil`, where a similar class exists? We already have `LocalJavaStreamingContext`; can this be combined?

Reply:
I'm not sure we can combine `LocalStreamingContext` and `LocalJavaStreamingContext`, as

1. LocalJavaStreamingContext relies on `junit`, whereas scala test suites rely on `scalatest`. I'm not sure both can be co-used.
2. LocalStreamingContext has to be a trait to co-use with SparkFunSuite (or its descendant) which many test suites are relying on.
2. It may bring more changes, as `LocalJavaStreamingContext` just initializes JavaStreamingContext in `setUp` whereas Scala test suites initialize StreamingContext when they need it - flexible with how to initialize StreamingContext. If we combine twos into one, the approach `LocalStreamingContext` is doing is the way to go, and we would need to modify existing Java UTs to reflect the change.

As I commented before that we no longer need StreamingTestUtils, I guess we're addressing first question now. Please let me know there's something we can apply more.

Reply:
OK that's fine. Can we keep them in the same package at least?

Reply:
Yes sure. I'll move the package. Thanks for guiding!